scripts/hostcfgd

```diff
@@ -383,6 +383,10 @@ class FeatureHandler(object):
         # On multi-ASIC device, creates systemd configuration file for each feature instance
         # residing in difference namespace.
         for feature_name in feature_names:
+            # As to syncd service, swss will monitor its stop and restart it, don't need systemd to auto-restart it.
+            if "syncd" in feature_name:
```

Comment:
Currently, the dependency between syncd and swss is defined in swss.sh.  Now the unnecessary auto-restart of syncd is triggered by systemd.  And the swss also use systemctl to re-start syncd.   Seems we could not differentiate these two auto-restart from syncd service's definition.   Any idea to check the dependency and block one at runtime?

Reply:
Why we let swss take charge of auto-restart of syncd here? That's because before swss start syncd and other services, it always flush DB at first. So that the syncd's restart could get a clear ASIC DB.

Reply:
Syncd and swss are designed to start/stop together due to they needs to be synchronized.

However, we do let the swss to take more control than syncd from day 1.

In this sense, I think syncd auto-stop is sufficient as long as swss can pick up the signal (by docker wait any) and restart both services.. @prsunny  do you have concerns?